tests/NSubstitute.Analyzers.Tests.Shared/NSubstituteVersion.cs

```diff
@@ -0,0 +1,8 @@
+namespace NSubstitute.Analyzers.Tests.Shared
+{
+    public enum NSubstituteVersion
+    {
+        NSubstitute4_2_2 = 1,
```

Comment:
I am following  similar naming convention as compiler team does in https://github.com/dotnet/roslyn/blob/d0d9a59a9c222e58e679645ff5fdc2c074c8335e/src/Compilers/CSharp/Portable/LanguageVersion.cs#L12
It doesnt look too elegant IMO but I didnt figure out anything better. Any other options?

Reply:
I think this is ok 👍